src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -159,12 +159,34 @@ public static Tag parseTag(String tag) throws ParseException {
     public static LocalDateTime parseAppointmentDateTime(String dateTime) throws ParseException {
         requireNonNull(dateTime);
         String trimmedDateTime = dateTime.trim();
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");
+        String[] dateTimeInfo = trimmedDateTime.split(" ");
+
+        if (dateTimeInfo.length != 2) {
+            throw new ParseException("Please ensure both Appointment date and time are entered in "
+                    + "dd-MM-yyyy HH:mm format.");
+        }
+
+        String dateInput = dateTimeInfo[0];
+        String timeInput = dateTimeInfo[1];
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
+        DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm");
+
+        LocalDate date;
+        LocalTime time;
+
         try {
-            return LocalDateTime.parse(trimmedDateTime, formatter);
+            date = LocalDate.parse(dateInput, dateFormatter);
         } catch (Exception e) {
-            throw new ParseException("Appointment date and time should be entered in dd-MM-yyyy HH:mm format!");
+            throw new ParseException("Appointment date is an invalid entry.");
```

Comment:
I think this command message can follow the one Hao Yu left in the other PR:  "Appointment date should be valid and entered in dd-MM-yyyy format!"